gobblin-service/src/test/java/org/apache/gobblin/service/modules/orchestration/DagManagerUtilsTest.java

```diff
@@ -209,6 +255,19 @@ public void testIsDagFinishedWithFinishRunningFailureOption() throws URISyntaxEx
     Assert.assertFalse(DagProcUtils.isDagFinished(dag));
   }
 
+  private List<Dag.DagNode<JobExecutionPlan>> randomizeNodes(Dag<JobExecutionPlan> dag) {
+    int size = dag.getNodes().size();
+    List<Dag.DagNode<JobExecutionPlan>> randomizedDagNodes = new ArrayList<>();
+
+    for (int i=0; i<size; i++) {
+      int index = rand.nextInt(size);
+      randomizedDagNodes.add(dag.getNodes().get(index));
+    }
+
+    return randomizedDagNodes;
+
+  }
```

Comment:
since this is the only place `rand` is used, suggest to make that a local var to the method.

Reply:
oh , forgot to clean this method. was trying to randomize using this way, but found something more elegant